deployments/kubernetes/chart/gitwebhookproxy/values.yaml

```diff
@@ -22,8 +22,8 @@ gitWebhookProxy:
     upstreamURL: "https://jenkins.tools.stackator.com"
     allowedPaths: "/github-webhook,/project"
     secret: ""
-    ignoredUsers: "stakater-user"
-    allowedUsers: "myuser"
+    ignoredUsers: ""
```

Comment:
This will be a breaking change for users(us atleast). Can you please let the default values be?